src/sockjs.coffee

```diff
@@ -8,6 +8,7 @@ events = require('events')
 fs = require('fs')
 webjs = require('./webjs')
 utils = require('./utils')
+urlParser = require("url")
```

Comment:
Can you use single quotes? Also I prefer to import core packages with the same name, so `url = require('url')`.